.idea/workspace.xml

```diff
@@ -20,11 +20,18 @@
   </component>
```

Comment:
May be worth taking a look at this workspace file and maybe not merge it. These changes looking system specific. But we have already talked about this. Just leaving a note